.github/workflows/unit-tests.yml

```diff
@@ -66,7 +66,25 @@ jobs:
 
       - name: install system requirements
         run: |
-          sudo apt-get update && sudo apt-get install libmysqlclient-dev libxmlsec1-dev lynx
+          sudo apt-get update && sudo apt-get install libmysqlclient-dev libxmlsec1-dev lynx openssl
+
+      # This is needed until the ENABLE_BLAKE2B_HASHING can be removed and we
```

Comment:
could you clarify more this. I think thats the reason in upstream the md4 is not used.

https://github.com/openedx/edx-platform/blob/master/.github/workflows/unit-tests.yml#L75